src/tree/context-dep.cc

```diff
@@ -179,6 +179,50 @@ void ContextDependency::Read (std::istream &is, bool binary) {
 }
 
 void ContextDependency::GetPdfInfo(const std::vector<int32> &phones,
```

Comment:
put a newline after ( and move things to the left so that you don't go so much above 80 characters.